packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -20,11 +22,12 @@ export interface Cip6MetadataFields {
    * 128 Characters Maximum, must be a valid URL
    */
   extSigUrl?: string;
+  // TODO: this is probably wrong, as CIP says it's 68 characters long. Find and validate the correct format.
   /**
    * the public Key for verification
    * optional, 68 Characters
    */
-  extVkey?: Hash16; // TODO: figure out if this is the correct type alias
+  extVkey?: Hash32ByteBase16;
```

Comment:
A 32 byte ed25519 verification key, rather than it's hash, bech32 encoded with prefix `vrf_vk`

https://github.com/cardano-foundation/CIPs/pull/15#discussion_r538230733

https://github.com/cardano-foundation/CIPs/blob/d5842cc0e58d08c4ba10f5a96d9b9ef14e78ff72/CIP-0005/README.md#keys

Reply:
`vrf_vk` prefix results in bech32 string length of 65 and CIP specifies this field length to be 68, which matches `poolmd_vk` prefix as per [this comment](https://github.com/cardano-foundation/CIPs/pull/15#discussion_r572736672). I think I'll make it accept both prefixes.

Reply:
How about PoolParameters? Ogmios has [this](https://github.com/input-output-hk/cardano-js-sdk/pull/145/files#diff-a9da9e5f1f8b2f32b8a4d95f189ec0afb6126e9b1839eef8a4aa1b29f512d0c8R41) as `Hash16` but since it's the same data maybe it's also not a hash but an actual key?

Reply:
`poolmd_vk` does indeed fit the bill, so let's just validate against this only.

> How about PoolParameters?

Yeah I agree it's likely not a hash, as we would have that in the name + we know the `Hash16` type has been used for non-hash values.